maintainers/maintainer-list.nix

```diff
@@ -5776,6 +5776,16 @@
     githubId = 20658981;
     name = "Jarosław Wygoda";
   };
+  jyooru = {
```

Comment:
Already added in https://github.com/NixOS/nixpkgs/pull/143831#pullrequestreview-799405913